### ISSUE TITLE
Failes to compile in Fedora 32

### ISSUE THREAD

**Issue by ormshaw** [KhronosGroup/OpenCL-ICD-Loader#99]
openCl failes to compile in Fedora 32 due to multiple definitions of ret_val in the loader_test.

I've attached
[diff.txt](https://github.com/KhronosGroup/OpenCL-Headers/files/4392503/diff.txt)

 the diffs to have a common ret_val, if this was what was intended, Although from a quick look at the code making each one static may also work

**Comment by kpet** [comment]
Thanks for flagging this. No good reason for `ret_val` or `ret_mem` to not be local variables. @ormshaw Do you want to propose a PR?

**Comment by alycm** [comment]
A similar issue came up a few days ago for the ICD: https://github.com/KhronosGroup/OpenCL-ICD-Loader/pull/98

**Comment by kpet** [comment]
Yes, just spotted it as well. This is the same issue (I've transfered this one from the headers to the ICD repo).

**Comment by bashbaug** [comment]
I just merged https://github.com/KhronosGroup/OpenCL-ICD-Loader/pull/101, so I think this issue can be closed now?
